Automattic/wp-calypso
# Remove multi year discount line from upsell

Related to #87243

## Proposed Changes

* Remove "multi year discount" line from Jetpack upsell

Before:
![CleanShot 2024-02-07 at 15 04 14@2x](https://github.com/Automattic/wp-calypso/assets/8419292/7f8aa8b4-6a75-44c6-b177-a627b36de447)

After change:
![CleanShot 2024-02-07 at 15 09 19@2x](https://github.com/Automattic/wp-calypso/assets/8419292/aa5de4d7-978c-4dae-9070-edde3d8698eb)


## Testing Instructions

* Using Calypso Live link, go to `/checkout/jetpack/jetpack_creator_yearly`
* Notice that "multi year discount" doesn't show up in the upsell

## Pre-merge Checklist
